src/domain/wallets/index.types.d.ts

```diff
@@ -75,4 +75,5 @@ type Wallet = {
 
 interface IWalletsRepository {
   findById(walletId: WalletId): Promise<Wallet | RepositoryError>
+  walletIdFromUsername(username: Username): Promise<WalletId | RepositoryError>
```

Comment:
This feels misplaced. First of all `Repository` should always return the whole object not just an `id` - so better would be:
```
wallet = repo.findWalletForUser
id = wallet.id
```

But actually the `WalletsRepository` should not know anything about usernames anyway. I think what we need is:
```
user = usersRepo.findByUsername(username)
defaultAccount = accountsRepo.findById(user.defaultAccountId)
walletId = deffaultAccount.walletIds[0]
```

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/449/commits/1e3fcd2bd79a2ea46b574cadd0119e3b95c120ca